src/platforms/apple/common/configuration/out-of-memory.mdx

```diff
@@ -1,24 +1,28 @@
 ---
 title: Out Of Memory
 sidebar_order: 10
-description: "Learn how to turn off Out Of Memory"
+description: "Learn how to turn off Out Of Memory tracking"
 ---
 
+<Note>
+This integration is only available before 8.0.0. We renamed it to <PlatformLink to="/configuration/watchdog-terminations/">watchdog terminations</PlatformLink> in the 8.0.0 release.
+</Note>
```

Comment:
Maybe this is obvious, but can we be more explicit about what we're talking about here? The SDK version? The iOS version?
```suggestion
<Note>

This integration is only available before version 8.0.0. It's now called <PlatformLink to="/configuration/watchdog-terminations/">Watchdog Terminations</PlatformLink> in the 8.0.0 release.

</Note>
```

Reply:
It's the SDK version, which I don't know how we call that in the docs. `sentry-cocoa`? `The Sentry SDK`?

Reply:
I think just saying "version 8.0.0 of the Sentry Apple SDK" is fine.